src/runtime/task/mod.rs

```diff
@@ -1,14 +1,18 @@
 use crate::runtime::execution::ExecutionState;
+use crate::runtime::task::clock::VectorClock;
 use crate::runtime::thread;
 use crate::runtime::thread::continuation::{ContinuationPool, PooledContinuation};
 use bitvec::prelude::*;
 use bitvec::vec::BitVec;
 use futures::{task::Waker, Future};
 use std::cell::RefCell;
+use std::cmp::PartialOrd;
```

Comment:
nit: remove

Reply:
Fixed.  Why doesn't clippy complain?

Reply:
Ah, because it's test code.  Guess I should run it with --cfg test.

Reply:
For the record, it's `cargo clippy --tests`.  TIL.